CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.18.1 (February 14th, 2024)
```

Comment:
The changelog notes below were not attached to any particular release @shane-ns1 

So I guess they will be release along side this one